src/network/protocol/messages/lib.zig

```diff
@@ -62,6 +66,7 @@ pub const Message = union(MessageTypes) {
             .sendcmpct => {},
             .feefilter => {},
             .filterclear => {},
+            .filteradd => {},
```

Comment:
and use deinit here

Reply:
Okay boss going over them all, will update it asap